cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -160,5 +166,48 @@ public boolean anyReadOnly(KafkaVersion kafkaVersion) {
         return result;
     }
 
+    /**
+     * Returns true if the supplied value is default to kafka in kafkaVersion version
+     * @param property name of the tested property
+     * @param value desired value
+     * @param kafkaVersion version of Kafka
+     * @return true if the desired value is default
+     */
+    @SuppressWarnings("checkstyle:Regexp")
+    public static boolean isValueOfPropertyDefault(String property, String value, KafkaVersion kafkaVersion) {
```

Comment:
Does this have to be `public`?

Reply:
Yes :( it is called from `KafkaBrokerConfigurationDiff`